suites.js

```diff
@@ -157,3 +187,36 @@ function createRulesSuite(rulesCount=50, indicatorIndexSize=10000) {
     scenarios,
   }
 }
+
+
+function createIMruleSuite({
+  rulesCount = 10,
+  indicatorIndexSize = 1000,
+  alertInterval = 1,
+  indicatorCount = 1,
+} = {}) {
+  const scenarios = [{
+    name: `${rulesCount} rules ${indicatorIndexSize} indicator index`,
+    alertInterval: `${alertInterval}m`,
+    alerts: rulesCount,
+    esSpec: '1 x 64 GB',
```

Comment:
It would be good to call out that these config are unused when specifying `esUrl` and `kbUrl`, best to be explicit whenever possible.